docs/learn/advanced/architecture.md

```diff
@@ -106,12 +107,26 @@ const sessionMessage = await createAppSessionMessage(messageSigner, sessionData)
 ### ClearNode Infrastructure
 
 Network infrastructure providing:
+
 - **Message routing**: Secure communication between participants
 - **Session management**: Application session lifecycle
 - **State coordination**: Off-chain state synchronization
 - **Network resilience**: Redundancy and failover support
 
-## Application Session Management
+## Application Session Protocol
```

Comment:
Hm, I'm not sure whether that is Protocol for App Sessions or for NitroRPC as a whole, we need to decide

Reply:
It feels unnatural that the only place we specify the version of NitroRPC protocol is in AppDefinition, and not in every RPC call. Moreover, are we going to change NitroRPC protocol at all, apart from "App session protocol"?
Thus, to avoid such logical problem, we can create a sub-protocol for AppSession communication only, decoupling it from NitroRPC.